bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/internal/EntityProvider.java

```diff
@@ -29,61 +31,75 @@
 import javax.ws.rs.ext.MessageBodyWriter;
 import javax.ws.rs.ext.Provider;
 
+import org.eclipse.smarthome.io.json.JsonBindingService;
 import org.osgi.service.component.annotations.Component;
-
-import com.google.gson.Gson;
-import com.google.gson.GsonBuilder;
+import org.osgi.service.component.annotations.Reference;
 
 /**
- * JSON reader/writer for Jersey using GSON.
+ * JSON reader/writer for JAX-RS that uses {@link JsonBindingService} to serialize and deserialize between
+ * Java objects and JSON.
  *
- * @author Simon Kaufmann - initial contribution and API
+ * @author Flavio Costa - Initial contribution and API
```

Comment:
You are not the initial author. Please add your name with an `@author` tag below Simon and state what you have done, i.e. changing the implementation to use the JSON abstraction layer.

Reply:
Originally, I copied the `GsonProvider` class, created the new `EntityProvider` and then deleted the old `GsonProvider`. It looks like Git does not consider the new class to be a new/independent implementation, so now I adjusted the `@author` tag accordingly, keeping Simon as the original contributor.